lib/artist.rb

```diff
@@ -0,0 +1,49 @@
+require 'pry'
```

Comment:
Since you've added `require 'pry' to the `environment.rb`, you shouldn't need it in individual Ruby class files.